src/lib/components/sections/Downloads.svelte

```diff
@@ -92,13 +93,14 @@
             <Dropdown {options} bind:selected={qt_version} />
           </div>
           <p class="text-lg md:text-2xl text-neutral">
-            Qt6 is recommended for most users, the reasoning can be found <a
+            Qt6 is recommended for most users, here are the <a
               href="https://docs.ankiweb.net/platform/windows/installing.html#qt5-vs-qt6"
+              aria-label="Anki Manual: Installing and Upgrading Anki on Windows"
               class="text-primary hover:opacity-80"
               target="_blank"
               rel="noopener noreferrer"
             >
-              here
+              reasons
```

Comment:
These will be removed after #8 has been merged. (I will resolve the merge conflict there.)